other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -46,13 +46,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    print(interface_data_holder.terminations)
</pre><p> </p><p>Maybe we should print the number of interfaces for the first termination, not all of them</p><p>Should be:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">print(f"Terminations found:  {interface_data_holder.terminations)}")
print(f"Interfaces for the first termination:  {interface_data_holder.get_interfaces_for_termination(0))}")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>